tests/treeTest.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+
+from coalahtml.tree import Node, Tree
+from coalib.coala_main import run_coala
+from coalib.output.printers.ListLogPrinter import ListLogPrinter
+from coalahtml.helper import parse_file_dict
+
+
+class TreeTest(unittest.TestCase):
+
+    def setUp(self):
+        log_printer = ListLogPrinter()
+        file_dict = run_coala(log_printer=log_printer,
+                              autoapply=False)[2]
```

Comment:
Yes. This was intentional. I need to operate on files and directories that are for real. I perform a check on node whether it's a file or a directory before creating a map. I could have created multiple files/directories using tempfile but that was not elegant IMO.